src/nbl/ext/ImGui/ImGui.cpp

```diff
@@ -469,14 +555,15 @@ namespace nbl::ext::imgui
 
 			logger->log(onError.data(), system::ILogger::ELL_ERROR);
 			assert(false);
+			return uint8_t(0); // silent warnings
 		};
 
 		// get & validate families' capabilities
 		families.id.transfer = requestFamilyQueueId(IQueue::FAMILY_FLAGS::TRANSFER_BIT, "Could not find any queue with TRANSFER_BIT enabled!");
 		families.id.graphics = requestFamilyQueueId(IQueue::FAMILY_FLAGS::GRAPHICS_BIT, "Could not find any queue with GRAPHICS_BIT enabled!");
```

Comment:
ask for them explicitly from the outside

Reply:
resolved